koku/api/common/pagination.py

```diff
@@ -306,20 +306,32 @@ def get_count(self, queryset):
         return len(queryset.get("data", [])[0].get("resource_ids", []))
```

Comment:
Reminder

Reply:
Updated to check for list before indexing